Source/JavaScriptCore/heap/Heap.cpp

```diff
@@ -128,6 +128,8 @@ size_t minHeapSize(HeapType heapType, size_t ramSize)
             static_cast<double>(Options::largeHeapSize()),
             ramSize * Options::smallHeapRAMFraction());
         return static_cast<size_t>(result);
+    } else if (heapType == HeapType::Medium) {
+        return static_cast<size_t>(Options::mediumHeapSize());
     }
```

Comment:
WebKit style is no braces for one line and no `else` if the previous `if` always returns.

Reply:
I think probably we should use switch to list up all possible heapType here.
```
switch (heapType) {
case HeapType::Large:
   ...
case HeapType::Medium:
   ...
case HeapType::Small:
   ...
default:
   RELEASE_ASSERT_NOT_REACHED();
   break;
}
```